dev-libs/boehm_gc/boehm_gc-7.6.4.recipe

```diff
@@ -0,0 +1,81 @@
+SUMMARY="The Boehm-Demers-Weiser conservative garbage collector"
+DESCRIPTION="The Boehm-Demers-Weiser conservative garbage collector can be \
+used as a garbage collecting replacement for C malloc or C++ new. It allows \
+you to allocate memory basically as you normally would, without explicitly \
+deallocating memory that is no longer useful. The collector automatically \
+recycles memory when it determines that it can no longer be otherwise \
+accessed."
+HOMEPAGE="http://www.hboehm.info/gc/"
+COPYRIGHT="1988, 1989 Hans-J. Boehm, Alan J. Demers
+	1991-1996 Xerox Corporation
+	1996-1999 Silicon Graphics
+	1999-2011 Hewlett-Packard Development Company, L.P."
+LICENSE="Boehm"
+REVISION="1"
+SOURCE_URI="https://github.com/ivmai/bdwgc/releases/download/v$portVersion/gc-$portVersion.tar.gz"
+CHECKSUM_SHA256="b94c1f2535f98354811ee644dccab6e84a0cf73e477ca03fb5a3758fb1fecd1c"
+SOURCE_DIR="gc-$portVersion"
+PATCHES="boehm_gc-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86_gcc2 ?x86"
+
+portVersionCompat="$portVersion compat >= 7"
+
+PROVIDES="
+	boehm_gc$secondaryArchSuffix = $portVersionCompat
+	lib:libgc$secondaryArchSuffix = $portVersionCompat
+	lib:libcord$secondaryArchSuffix = $portVersionCompat
```

Comment:
```diff
 PROVIDES="
 	boehm_gc$secondaryArchSuffix = $portVersionCompat
-	lib:libgc$secondaryArchSuffix = $portVersionCompat
-	lib:libcord$secondaryArchSuffix = $portVersionCompat
+	lib:libcord$secondaryArchSuffix = $libcordVersionCompat
+	lib:libgc$secondaryArchSuffix = $libgcVersionCompat
 	"
```
(`$portVersion` is only OK for the first line. For the `lib:` and `devel:` lines we have different values.)
(BTW, we like to sort alphabetically the `lib:` lines.)